gulpfile.js

```diff
@@ -99,13 +100,13 @@ gulp.task('clean-bin-dist', function() {
 //move binary to bin, but don't keep it - it's in .gitignore; only *.zips are kept
 //whatever the platform, the user calls slyft, not slyft.mac etc.
 gulp.task('package-binary', function() {
-  return gulp.src(['./slyft-cli', './slyft-cli.exe'], { base: '.' })
+  return gulp.src(['./slyft', './slyft.exe'], { base: '.' })
   .pipe(gulp.dest('bin'))
 });
 
 gulp.task('dist', function() {
   return gulp.src('./bin/**/*', { base: './bin' })
-  .pipe(zip(pkg.name + '-' + pkg.version + '-' + platform + '.zip'))
+  .pipe(zip(pkg.name + '-' + pkg.version + '-' + platform + '-' + arch + '.zip'))

```

Comment:
need to add a third string to the 'clean-bin-dist' task that handles the arch string; that way we only have one dist binary per platform at a time, so `git clone --depth 1` doesn't take too long